### PR TITLE
Issue #36, Error handling on Uint64.fromUnit with negative value.

### PR DESCRIPTION
**Pull request by rg911**
Couple of tslint issues fixed on MosaicProperty as well

### CODE REVIEW

File: src/model/UInt64.ts [hunk 1]
```diff
@@ -37,6 +37,9 @@ export class UInt64 {
      * @returns {UInt64}
      */
     public static fromUint(value: number): UInt64 {
+        if (value < 0) {
```
Comment (evias) [inline src/model/UInt64.ts]: This check would better be done in the class constructor.

Also the error message contains a typo. Either of 'UInt' or 'Unsigned Integer' would be fine 👍

File: test/model/UInt64.spec.ts [hunk 2]
```diff
@@ -64,6 +64,12 @@ describe('Uint64', () => {
         expect(uint64Compact).to.be.equal(51110867862);
     });
 
+    it('should fromUnit throw exception with negative unit value', () => {
```
Comment (evias) [inline test/model/UInt64.spec.ts]: Same typo in UInt.

File: src/model/UInt64.ts (hunk 1 above)
Comment (rg911) [reply src/model/UInt64.ts]: @evias 32bit unsigned int capped at 4294967295 and -1 >>> 0 === 4294967295. Constructor only takes UInt arrays, how does it differentiate -1 and real 4294967295?

Comment (evias) [reply src/model/UInt64.ts]: I agree @rg911, you are right. 
I believe adding it in fromUInt makes sense like this then :+1:

Comment (rg911) [reply src/model/UInt64.ts]: 👍

Comment (rg911) [reply src/model/UInt64.ts]: More clean way is to add the check in NEM2-library I'd thought?

**Pull request merged**